camera_matrix:
- - 1587.6372248250916
  - 0.0
  - 182.25520906366071
- - 0.0
  - 1550.6038808641108
  - 98.6761380922422
- - 0.0
  - 0.0
  - 1.0
dist_coeff:
- - -5.264616311200455
  - 662.9796415915182
  - -0.00874792214294252
  - -0.119195212886408
  - -6.822298751135376
mean_error: 0.08911316207166718
new_camera_matrix:
- - 1730.2957763671875
  - 0.0
  - 172.50839423844354
- - 0.0
  - 1717.735107421875
  - 100.31217476009715
- - 0.0
  - 0.0
  - 1.0
roi:
- 0
- 0
- 319
- 239
rotation_matrix:
- - -0.009529549434871876
  - 0.9715363523929939
  - 0.23669876143842644
- - -0.9998711863840255
  - -0.0062006996037629845
  - -0.014804119812589472
- - -0.012915042646849548
  - -0.23680934800666267
  - 0.9714702951557963
rotation_vector:
- - -0.17686249758675263
- - 0.19885712233794525
- - -1.5705398647848816
translation_vector:
- - -2.2927192263160414
- - 2.0807202583768083
- - 140.4273866349936
